core/src/main/res/raw/ayp_youtube_player.html

```diff
@@ -166,5 +169,51 @@
       player.toggleFullscreen();
     }
 
+    function cuePlaylist(videos, index, startSeconds) {
+      player.cuePlaylist(videos, index, startSeconds);
+      YouTubePlayerBridge.sendVideoId(videos[index]);
```

Comment:
https://github.com/PierfrancescoSoffritti/android-youtube-player/pull/1006/files can only handle a single string, so I wasn't sure if I should serialize the arguments & deserialize again. I figured this would be simpler.